CMakeLists.txt

```diff
@@ -86,11 +86,10 @@ TensileCreateLibraryFiles(
     "${CMAKE_CURRENT_SOURCE_DIR}/yaml/${MIOPEN_TENSILE_SRC}"
     "${CMAKE_CURRENT_BINARY_DIR}/lib/miopentensile"
     ARCHITECTURE ${Tensile_ARCHITECTURE}
-    VAR_PREFIX MIOPEN_TENSILE
```

Comment:
I removed this extra prefix during building. But it doesn't help.

Reply:
Additional fix coming from Tensile side, https://github.com/ROCmSoftwarePlatform/Tensile/pull/1106

When that is merged in, you will have to update the commit tag for Tensile